services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -35,37 +35,52 @@ export const ReleaseDialog: React.FC<ReleaseDialogProps> = (props) => {
     const dialog =
         props.version !== -1 ? (
             <div>
-                <Dialog fullWidth={true} maxWidth="md" open={props.app !== ''} onClose={setClosed}>
-                    <AppBar sx={{ position: 'relative' }}>
-                        <div className={classNames('release-dialog-header', props.className)}>
-                            Application: {props.app} version: {props.version}
-                        </div>
+                <Dialog
+                    className={classNames('release-dialog', props.className)}
+                    fullWidth={true}
+                    maxWidth="md"
+                    open={props.app !== ''}
+                    onClose={setClosed}>
+                    <div className={classNames('release-dialog-app-bar', props.className)}>
                         <div className={classNames('release-dialog-message', props.className)}>
                             {props.release.sourceMessage}
                             <span className={classNames('release-dialog-commitId', props.className)}>
                                 {props.release.sourceCommitId}
                             </span>
+                            <span className={classNames('release-dialog-close', props.className)}>
+                                <Button onClick={setClosed}>
+                                    <svg
+                                        width="20"
+                                        height="20"
+                                        viewBox="0 0 20 20"
+                                        fill="none"
+                                        xmlns="http://www.w3.org/2000/svg">
+                                        <path
+                                            d="M1 1L19 19M19 1L1 19"
+                                            stroke="white"
+                                            strokeWidth="2"
+                                            strokeLinecap="round"
+                                        />
+                                    </svg>
+                                </Button>
+                            </span>
                         </div>
                         <div className={classNames('release-dialog-createdAt', props.className)}>
                             {!!props.release.createdAt && (
                                 <div>
-                                    <span>{'Release date ' + props.release.createdAt.toLocaleDateString()}</span>
-                                    <span className={classNames('release-dialog-hour', props.className)}>
-                                        {'Hour ' + props.release.createdAt.toLocaleTimeString()}
-                                    </span>
+                                    {'Release date ' +
+                                        props.release.createdAt.toISOString().split('T')[0] +
```

Comment:
```suggestion
                                        props.release?.createdAt.toISOString().split('T')[0] +
```
please use chaining when accessing nested attributes